.github/CODEOWNERS

```diff
@@ -76,6 +76,7 @@
 /rust/send_http_get/ @dfinity/growth
 /rust/send_http_post/ @dfinity/growth
 /rust/simd/ @dfinity/execution
+/rust/stake_neuron_from_cli @dfinity/governance
```

Comment:
@marc0olo Is there a way to add our team to this repo, with the same configuration as in dfinity/ic?

Reply:
I am not sure how that is configured. probably a question for @dfinity/idx 🤔 

isn't adding your team like this sufficient?

Reply:
<img width="1133" height="65" alt="image" src="https://github.com/user-attachments/assets/b3be9ded-aa9a-4280-b3b8-e154531fde6f" />

Reply:
I gave `@dfinity/governance-team` write access to this repo. Do add `-team`.

Reply:
Thanks!